api/managementpb/backup/errors.proto

```diff
@@ -0,0 +1,20 @@
+syntax = "proto3";
+
+package backup.v1beta1;
+
+option go_package = "api/managementpb/backup;backupv1beta1";
+
+// ErrorCode is a set of specific errors that are not present in the standard set of errors
+// and returned in the details field of the response.
+enum ErrorCode {
+  ERROR_CODE_INVALID = 0;
+  ERROR_CODE_XTRABACKUP_NOT_INSTALLED = 1;
+  ERROR_CODE_INVALID_XTRABACKUP = 2;
+  ERROR_CODE_INCOMPATIBLE_XTRABACKUP = 3;
+  ERROR_CODE_INCOMPATIBLE_TARGET_MYSQL = 4;
+}
+
+// Error represents error message returned in the details field in the response.
+message Error {
+  ErrorCode code = 1;
```

Comment:
Let's rename this field to make it more clear that this is not a gRPC error code.

Reply:
not sure, since it is located in the backups package and also has a comment that mentions where this error should be returned.

Reply:
IMHO that's can be a little bit confusing:
```
{
    "error": "something happened",
    "code": code_precondition_failed,
    "message": "something happened",
    "details": [
        {
            "@type": "type.googleapis.com/backup.v1beta1.Error",
            "code": "ERROR_CODE_INVALID_XTRABACKUP"
        }
    ]
}
```

Reply:
Ha, I just noticed, why are we returning error text names, not just codes?

Reply:
it has a `@type` field indicating from where it came from e.g. from package `backup.v1beta1`

Reply:
> Ha, I just noticed, why are we returning error text names, not just codes?

well looks like it is gonna be just an integer :) because I'm using enum.

Reply:
So on FE, we need to compare strings to understand what error? It looks like have integer code is simpler.

@fabio-silva what do you think?

Reply:
Data race in comments)

Reply:
@artemgavrilov we had a commenting race condition :D

Reply:
> So on FE, we need to compare strings to understand what is the error? It looks like have integer code is simpler.
> 
> 
> 
> @fabio-silva what do you think?

I agree it's simpler the integer way, even though we also have string enums in the FE.